src/core/Effect.cpp

```diff
@@ -146,19 +143,19 @@ Effect * Effect::instantiate( const QString& pluginName,
 
 
 
-void Effect::checkGate( double _out_sum )
+void Effect::checkGate(double _out_sum)
```

Comment:
You can add these spaces back in. Style change only those lines you actively worked on.